apps/src/javalab/JavalabView.jsx

```diff
@@ -75,7 +75,8 @@ class JavalabView extends React.Component {
 
   state = {
     loading: true,
-    loadSuccess: null
+    loadSuccess: null,
+    isDarkMode: false
```

Comment:
would it be worth putting `isDarkMode` in redux so we don't have to pass it down to other components? We'll be adding more components as we add more features that will need to know the theme.